disperser/cmd/batcher/flags/flags.go

```diff
@@ -154,6 +154,13 @@ var (
 		EnvVar:   common.PrefixEnvVar(envVarPrefix, "MAX_NUM_RETRIES_PER_BLOB"),
 		Value:    2,
 	}
+	TargetNumChunksFlag = cli.UintFlag{
```

Comment:
Comment what's the use case of this flag?

Reply:
Added.